frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py

```diff
@@ -47,7 +47,7 @@ def krb_straighten_username(krb_remote_user):
     if not re.match(r"^[\w.-]+$", username):
         return None
 
-    for pattern in app.config.get("KRB5_USER_DENYLIST_PATTERNS", []):
+    for pattern in app.config.get("KRB5_USER_DENYLIST_PATTERNS") or []:
```

Comment:
You could probably do `app.config.get("KRB5_USER_DENYLIST_PATTERNS", [])` but it's not even worth changing.

Reply:
Nah, that's what we had before..?

Reply:
/me dumb 
I understand the problem now.

Reply:
np, thank you for the review!